cylc/flow/suite_files.py

```diff
@@ -678,10 +678,10 @@ def remote_clean(reg, platform_names, timeout):
             if ret_code:
                 # Try again using the next platform for this install target:
                 this_platform = platforms.pop(0)
-                exc = TaskRemoteMgmtError(
+                excn = TaskRemoteMgmtError(
```

Comment:
mypy doesn't like vars called `exc` unless they are in `except:` branches.

Reply:
Huh, that's an interesting one

Reply:
I think it's because there's an earlier line `except Exception as exc:`. https://github.com/python/mypy/issues/5080